### PR TITLE
[MM-17379] Fixed Management text

### PR DESCRIPTION
**Pull request by hahmadia**
#### Summary
This PR fixes the text displayed under the management heading in `System Console > Teams`.
Edited appearing text to match what is shown on `User Configuration > Team Membership` (Managed in `components/admin_console/system_user_detail/team_list/team_row.jsx`) to be more consistent.

 
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-17379

#### Fix
Kapture 2019-10-24 at 17 53 00

### CODE REVIEW

File: components/admin_console/team_channel_settings/team/list/team_row.jsx [hunk 1]
```diff
@@ -19,6 +19,32 @@ export default class TeamRow extends React.Component {
         const {team, onRowClick} = this.props;
         onRowClick(team.id);
     }
+
+    renderManagementMethodText = () => {
+        const {team} = this.props;
+        if (team.group_constrained) {
+            return (
+                <FormattedMessage
+                    id={'admin.team_settings.team_row.managementMethod.groupSync'}
```
Comment (jespino) [inline components/admin_console/team_channel_settings/team/list/team_row.jsx]: This shouldn't be `{'admin.team_settings.team_row.managementMethod.groupSync'}` use instead just `id='admin.team_settings.team_row.managementMethod.groupSync'`

File: components/admin_console/team_channel_settings/team/list/team_row.jsx [hunk 2]
```diff
@@ -19,6 +19,32 @@ export default class TeamRow extends React.Component {
...
+                    defaultMessage={'Group Sync'}
```
Comment (jespino) [inline components/admin_console/team_channel_settings/team/list/team_row.jsx]: Same here

File: components/admin_console/team_channel_settings/team/list/team_row.jsx [hunk 3]
```diff
@@ -19,6 +19,32 @@ export default class TeamRow extends React.Component {
...
+                />
+            );
+        } else if (team.allow_open_invite) {
+            return (
+                <FormattedMessage
+                    id={'admin.team_settings.team_row.managementMethod.anyoneCanJoin'}
```
Comment (jespino) [inline components/admin_console/team_channel_settings/team/list/team_row.jsx]: Same here and in the next line

File: components/admin_console/team_channel_settings/team/list/team_row.jsx [hunk 4]
```diff
@@ -19,6 +19,32 @@ export default class TeamRow extends React.Component {
...
+                    defaultMessage={'Anyone Can Join'}
+                />
+            );
+        }
+        return (
+            <FormattedMessage
+                id={'admin.team_settings.team_row.managementMethod.inviteOnly'}
```
Comment (jespino) [inline components/admin_console/team_channel_settings/team/list/team_row.jsx]: Same here and in the next line

File: components/admin_console/team_channel_settings/team/list/team_row.jsx (hunk 1 above)
Comment (hahmadia) [reply components/admin_console/team_channel_settings/team/list/team_row.jsx]: Oops! Not sure how I didn't catch this. Thanks! Fixed 😄

**Pull request merged**